### ISSUE TITLE
Settings Tab Empty?

### ISSUE THREAD

**Issue by cschlisner** [krojew/OpenRGB-Ambient#12]
Hi I wanted to try this out as I was having a lot of issues with the ambient effect in the "OpenRGB Effects plugin". 

I am on OpenRGB version 0.9 trying to use the 2.4 release, and I see this when I go to the Ambient tab:
![image](https://github.com/krojew/OpenRGB-Ambient/assets/3262210/2b9d2603-75ac-4236-bbe7-58f7e149b326)

There's no options for assigning devices -- am I missing something or doing anything incorrectly? 

Thanks

**Comment by krojew** [comment]
What device are you using?

**Comment by cschlisner** [comment]
Here's my Device list: 
![image](https://github.com/krojew/OpenRGB-Ambient/assets/3262210/0f97748c-6abd-494c-b6e3-2520d610028e)

I Have 1 4pin and 1 3pin header on the mother board, and the arduino is set up as a TPM2 device. I am on Windows 10 if that's what you meant

**Comment by cschlisner** [comment]
Yes everything works fine/as expected with other plugins and can be controlled directly

**Comment by highonskooma** [comment]
Any news on this? I'm facing the same issue.
I'm on windows 11 with openRGB version 0.9 and ambient 2.4.0.

**Comment by krojew** [comment]
I'm sorry, but without the actual hardware, I can only guess what's going on.. Do the devices have "Direct" mode available in the "Devices" tab?

**Comment by highonskooma** [comment]
I have a gigabyte B650M gaming x ax motherboard with 2 3-pin and 1 4-pin rgb connectors. 
Some of the fans are controlled by a corsair lighting node core controller, I also have 2 dominator ram sticks, a 7800 xt with it's own rgb connection, All devices except for the gpu are set to direct mode on the devices tab of openrgb, gpu is set to "external control". I've tried restarting openrgb and re-adding the plugin and nothing shows up.

**Comment by krojew** [comment]
The only thing which can exclude a device from appearing on the settings tab is the lack of "Direct" mode. Therefore, without the actual hardware, I cannot do any investigation.

**Comment by malthis12** [comment]
Has there been any update on this? I'm encountering the same problem on Windows 11 with OpenRGB 0.9 and 0.8 with the plugin 2.4.0 and 2.3.1.

**Comment by krojew** [comment]
Can you confirm your device supports direct mode?

**Comment by malthis12** [comment]
After disabling the VisualMap plugin the devices appear, however even after selecting them nothing happens.

**Comment by krojew** [comment]
After selecting a device, you need to configure zones using the sliders.

**Comment by malthis12** [comment]
Thank you, that works. However, it's impossible to get the plugin to work with VisualMap.
